src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,129 @@
+package duke;
+
+import java.time.LocalDate;
+
+class Ui {
+
+	/**
+	 * displays the welcome message
+	 */
+	static void welcome() {
+		System.out.println("    ____________________________________________________________");
+		System.out.println();
+		System.out.println("      Hello! I'm Duke");
+		System.out.println("      What can I do for you?");
+		System.out.println("    ____________________________________________________________");
+		System.out.println();
+	}
+
+	/**
+	 * displays the message when an invalid input is given
+	 */
+	static void invalidInput() {
```

Comment:
Perhaps the name of the method should be a verb instead? I noticed some of your other methods have the same issue.